### DISCUSSION TITLE
Nonpolarizable Li Cathode for Graphite anode Simulations in 3E cells

### DISCUSSION THREAD

**Discussion by mpegis** [pybamm-team/PyBaMM#1892] [Q&A]
Hi All,

Sorry for stupid questions here- very new to battery simulations but have been doing experimental electrochemistry for about a decade....

Ive been working on fast charge fundamentals on graphite anodes lately, and have been using 3E cells to do this. Its basically a half cell (Li counter : Separator : Graphite working electrode) with an embedded reference to control voltage (Voltage = Voltage graphite  - reference voltage). As such, the polarization of the Li counter is not included in my fastcharge measurements. 

I have gone through most of the training modules and have successfully simulated in the full cell SPME model and also the half cell model. However, I wanted to hear your thoughts to make sure my ideas concerning implementation of a Li:Sep:Graphite cell for simulations is accurate.....

My basic idea was to setup a full cell model using Li as the anode from the default settings provided by xu
(PyBaMM/pybamm/input/parameters/lithium_ion/negative_electrodes/li_metal_Xu2019/)

Furthermore, since polarization of this anode can be ignored in my 3E setup, can i just simply set my exchange current density to enormously high values to emulate a nonpolarizable electrode? I was also thinking of setting the transfer coefficients to 1, but wasnt sure if that could cause some problems (e.g. does the package assume an alpha of 1 would be a tafel slope of 60 mV/dec in current density in one direction, but a tafel slope of infinity in the other?).

Any advice would be great. Let me know what you think!

Best,

MP

**Comment by brosaplanella** [comment]
Hi MP! Sorry for the delay in the response. I am not very familiar with the polarisation of the electrode. From what I understand, polarisation is caused by diffusion effects (which do not apply to the lithium anode), while the reaction coefficient causes the reaction overpotential which is another effect. If I understand correctly what you want to do, it should suffice to use the half-cell (or lithium metal) model as it comes.
